nixops/resources/__init__.py

```diff
@@ -103,13 +102,20 @@ def get_type(cls) -> str:
     _created_event: Optional[Event] = None
     _destroyed_event: Optional[Event] = None
     _errored: Optional[bool] = None
+
+    # While this looks like a rookie mistake where the list is going  get shared
+    # across all class instances it's not... It's working around a Mypy crash.
+    #
+    # We're overriding this value in __init__.
+    # It's safe despite there being a shared list on the class level
```

Comment:
Can you include a link to a bug ref, and/or a version number, or some more specific information for future spelunkers to explain exactly what code is related to this, and what they can delete once it is fixed?